misk/src/main/kotlin/misk/web/dashboard/AdminDashboardProtobufDocUrlPrefix.kt

```diff
@@ -0,0 +1,5 @@
+package misk.web.dashboard
+
+data class AdminDashboardProtobufDocUrlPrefix(
+  val url: String
+) : ValidWebEntry(url_path_prefix = url)
```

Comment:
Given the prefix is a full url (ie. https://internalprotoservice.corp.co/protos/), I'm not sure that fits with the assumptions of `ValidWebEntry`. Can you add a test to confirm this binding ends up in the WebActionMetadataAction as expected?

Reply:
I based this on some other examples that are using `ValidWebEntry` to validate a url: https://github.com/cashapp/misk/blob/b8a0c02e87ea94a704876e854134f5a47421ef5f/misk/src/main/kotlin/misk/web/actions/WebActionEntry.kt#L16-L19

I've added a test to `WebActionMetadataActionTest` to ensure that the value is being set as expected.